apis/policies/v1alpha2/admissionpolicy_webhook.go

```diff
@@ -49,6 +49,12 @@ var _ webhook.Defaulter = &AdmissionPolicy{}
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *AdmissionPolicy) Default() {
 	admissionpolicylog.Info("default", "name", r.Name)
+	if r.Spec.PolicyServer == "" {
+		// TODO?: this could be configurable through an argument on the
+		// controller. Whether or not to default policy server on
+		// admission policies, and to what policy server
```

Comment:
By being an argument on the controller the way this webhook behaves is not configurable at the helm chart level (where we decide to create the default policy server as well).

cc/ @kubewarden/kubewarden-developers